ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -110,7 +110,7 @@ export class AccountCreationInstitution extends React.Component<Props, State> {
       const details = await institutionApi().getPublicInstitutionDetails();
       this.setState({
         loadingInstitutions: false,
-        institutions: fp.sortBy( institution => institution.displayName, details.institutions)
+        institutions: fp.sortBy( institution => institution.displayName.trim(), details.institutions)
```

Comment:
To prevent new-user confusion.

I chose NOT to trim for the sorting function on the Inst Admin page, to make the problem visible to admins.